cap/modules/auth/views.py

```diff
@@ -207,3 +209,43 @@ def get_oauth_profile(name, token=None, client=None):
     except AttributeError:
         res_json = {}
     return res_json
+
+
+def get_oidc_user_groups(email):
+    """Retrieves the logged-in user's groups through CERN OIDC."""
+    id_api = 'https://authorization-service-api.web.cern.ch/api/v1.0/Identity'
+    token_api = 'https://auth.cern.ch/auth/realms/cern/api-access/token'
+
+    # get a new token for the client
+    token_resp = requests.post(
+        url=token_api,
+        headers={"Content-Type": "application/x-www-form-urlencoded"},
+        data={'client_id': current_app.config.get('CERN_CLIENT_ID'),
+              'client_secret': current_app.config.get('CERN_CLIENT_SECRET'),
+              'grant_type': 'client_credentials',
+              'audience': 'authorization-service-api'}
+    )
+    if token_resp.ok:
+        token = token_resp.json()['access_token']
+    else:
+        return
+
+    # get the user's uuid
+    id_resp = requests.get(
```

Comment:
Should we just use one request to the authorization service:
`/api/v1.0/Identity/current/groups`